openpathsampling/storage/storage.py

```diff
@@ -131,15 +129,26 @@ def _initialize(self):
         # Set global attributes.
         setattr(self, 'title', 'OpenPathSampling Storage')
 
-        self.set_caching_mode()
+        # backwards compatibility
+        self.cvs = self.attributes
 
-        # since we want to store stuff we need to finalize stores that have not
-        # been initialized yet
-        self.finalize_stores()
+        self.set_caching_mode()
 
     def _restore(self):
         self.set_caching_mode()
 
+        if hasattr(self, 'cvs'):
+            logger.info('Opening an old version that handles CVs differently. '
+                        'You cannot extend this file, only savely read it.')
```

Comment:
"Safely" not "savely" (or just remove the word entirely)

Reply:
thanks...